build/tools/Automation/DotNet/Models/FileObj.cs

```diff
@@ -0,0 +1,23 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+using Newtonsoft.Json;
+
+namespace Microsoft.Oryx.Automation.DotNet.Models
+{
+    public class FileObj
```

Comment:
nit: I'm typically opposed to including `Obj` in the name of a class since it's already implied that it'll be an object once instantiated, but if this makes more sense for you, then I'm OK keeping it this way

(this comment applies for all class with the `Obj` suffix)

Reply:
So `File` belongs to belongs to `System.IO.File`. I'll just have to specify the namespace to differentiate. What are you thoughts on that?

Reply:
I think if we set up the classes correctly (_i.e._, if we contain all of the I/O calls within their own `FileService` class), then I would be optimistic we wouldn't use a `File` object in the same class that has a `using System.IO` reference.

If that's not the case, then we could defer to using the `Metadata` suffix instead of `Obj`, which should make sense in the context of these classes you've created.